client/blocks/comments/post-comment-content.jsx

```diff
@@ -48,6 +48,7 @@ class PostCommentContent extends React.Component {
 					<Emojify>
 						<div
 							className="comments__comment-content"
+							data-e2e-comment={ this.props.content.slice( 3, this.props.content.length - 5 ) }
```

Comment:
If I use only `this.props.content` then comment text has `<p>` tag around it. Is there a better way to get content?